src/pipecat/processors/filters/identity_filter.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2024, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+from typing import Awaitable, Callable
+
+from pipecat.frames.frames import Frame, SystemFrame
+from pipecat.processors.frame_processor import FrameDirection, FrameProcessor
+
+
+class IdentityFilter(FrameProcessor):
+    """This filter just lets all frames go through, so it basically performs no
+    function. It can be useful when testing `ParallelPipeline` to create
+    pipelines that pass through frames (no frames should be repeated).
+
+    """
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    #
+    # Frame processor
+    #
+
+    async def process_frame(self, frame: Frame, direction: FrameDirection):
```

Comment:
We should decide where we want docstrings. Including everywhere seems overkill. If you want one here, it would be something like:
```
"""Process an incoming frame by passing it through unchanged."""
```

Reply:
Added